sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -598,6 +598,17 @@ object SQLConf {
       .bytesConf(ByteUnit.BYTE)
       .createOptional
 
+  val ADAPTIVE_MAX_SHUFFLE_HASH_JOIN_LOCAL_MAP_THRESHOLD =
+    buildConf("spark.sql.adaptive.maxShuffledHashJoinLocalMapThreshold")
+      .doc("Configures the maximum size in bytes per partition that can be allowed to build " +
+        "local hash map. If this value is not smaller than " +
```

Comment:
I would say:
- "If this value is not smaller than" => "If this value is larger than"
- "all the partition size are not larger than this config" => "all the partition size are smaller than this config"

Reply:
I thought about this but the formula used equal, that said it would be `If this value is larger than or equal to xxx`. Do you think it's better ?